server/client_manager.py

```diff
@@ -2107,3 +2148,30 @@ def get_multiclients(self, ipid=-1, hdid=""):
 
     def get_mods(self):
         return [c for c in self.clients if c.is_mod]
+        
+    class Battle_char:
+        def __init__(self, client, arg):
+            with open(
+                "storage/battlesystem/battlesystem.yaml", "r", encoding="utf-8"
```

Comment:
shouldn't the object itself be independent of the file saving/loading? I don't think it's a good idea to have this in the __init__ function

Reply:
Done!